### PR TITLE
Add multiple helpers to templates

### PR DESCRIPTION
**Pull request by Moppa5**
#### What are the main changes in this PR?
Helpers include this far: 
- Paragraph
- Text / div field
- Section title
- Tooltip
- Spinner
- Button
- Success/Failure

Check that the created components are ok and they meet our requirements. 

##### Why are we doing this? Any context or related work?
See #538

### CODE REVIEW

File: lib/postbox/template.php [hunk 1]
```diff
@@ -23,5 +24,84 @@ public static function error_paragraph( $message ) {
...
+      );
+    }
+
+    /**
+     * Display spinner image. In default it's hidden.
+     * @param string $id Id for this spinner.
+     * @return \Seravo\Postbox\Component Spinner component.
+     */
+    public static function spinner( $id ) {
+      return Component::from_raw('<div id="' . $id . '"><img src="/wp-admin/images/spinner.gif" hidden></div>');
```
Comment (JoosuaKoskinen) [inline lib/postbox/template.php:84]: Could you remove the `hidden` and add option to add classes. Then you can make hidden spinner by giving `hidden` class and visible spinner by giving it nothing. Currently you can only make hidden spinners.

File: lib/postbox/template.php [hunk 2]
```diff
@@ -23,5 +24,84 @@ public static function error_paragraph( $message ) {
...
+        $color = 'green';
+        $msg = sprintf('<b>%s</b>', __('Success!', 'seravo'));
+      } else {
+        $color = 'red';
+        $msg = sprintf('<b>%s</b>', __('Failure!', 'seravo'));
+      }
+
+      return Component::from_raw('<p style="color: ' . $color . ';">' . $msg . '</p>');
+
+    }
```
Comment (JoosuaKoskinen) [inline lib/postbox/template.php:104]: Nothing wrong with this but maybe remove the first and last empty lines for consistency.

**Review by JoosuaKoskinen — CHANGES_REQUESTED** [review]

**Comment by JoosuaKoskinen** [comment]
Feel free to merge after resolving change requests.

File: lib/postbox/template.php (hunk 1 above)
Comment (Moppa5) [reply lib/postbox/template.php]: Added $class = 'spinner' default parameter and the class attribute for the div.

File: lib/postbox/template.php (hunk 2 above)
Comment (Moppa5) [reply lib/postbox/template.php]: Removed. The first empty line was due to the condition block as I like it better. However, they are both now removed.

**Pull request merged**